contao/languages/en/tl_metamodel_dcasetting_condition.php

```diff
@@ -42,11 +42,12 @@
 $GLOBALS['TL_LANG']['tl_metamodel_dcasetting_condition']['addall']     = array('Add all', 'Add all attributes to input screen');
 
 // filter condition names.
-$GLOBALS['TL_LANG']['tl_metamodel_dcasetting_condition']['conditionnames']['conditionor']              = 'OR';
-$GLOBALS['TL_LANG']['tl_metamodel_dcasetting_condition']['conditionnames']['conditionand']             = 'AND';
-$GLOBALS['TL_LANG']['tl_metamodel_dcasetting_condition']['conditionnames']['conditionpropertyvalueis'] = 'Property value is';
-$GLOBALS['TL_LANG']['tl_metamodel_dcasetting_condition']['conditionnames']['conditionpropertyvisible'] = 'Property is visible';
-$GLOBALS['TL_LANG']['tl_metamodel_dcasetting_condition']['conditionnames']['conditionnot']             = 'Not';
+$GLOBALS['TL_LANG']['tl_metamodel_dcasetting_condition']['conditionnames']['conditionor']                      = 'OR';
+$GLOBALS['TL_LANG']['tl_metamodel_dcasetting_condition']['conditionnames']['conditionand']                     = 'AND';
+$GLOBALS['TL_LANG']['tl_metamodel_dcasetting_condition']['conditionnames']['conditionpropertyvalueis']         = 'Property value is';
+$GLOBALS['TL_LANG']['tl_metamodel_dcasetting_condition']['conditionnames']['conditionpropertymultiplevalueis'] = 'Property values are';
```

Comment:
As far as I have read in a quick glance, you are checking for "first hit is ok".
Therefore this language string should rather be: `Property values contain any of` than `Property values are`.

What do you think?